api/src/db/config/dynamoDbConfig.ts

```diff
@@ -22,11 +23,23 @@ export const unmarshallOptions = {
 
 export const dynamoDbTranslateConfig = { marshallOptions, unmarshallOptions };
 
-export const migrateData = (data: any): any => {
+export const migrateData = (data: any, logger: LogWriterType): any => {
+  const logId = logger.GetId();
   const dataVersion = data.version ?? CURRENT_VERSION;
   const migrationsToRun = Migrations.slice(dataVersion);
   const migratedData = migrationsToRun.reduce((prevData: any, migration: MigrationFunction) => {
-    return migration(prevData);
+    try {
+      logger.LogInfo(
+        `Database Migration - Id:${logId} - Upgrading ${data.user.id} from ${prevData.version} to ${
+          Number(prevData.version) + 1
```

Comment:
pebble - we are not always migrating by an increment of 1. Once the once time migration occurs, the migrations will occur when they log onto the system. The correct value should be `Migrations.length + 1`

Reply:
Good catch. Missed this update.